docs/getting-started/what-is-dotnet.md

```diff
@@ -55,11 +55,11 @@ Although the .NET Framework and .NET Core target different platforms and represe
 
 A number of development technologies depend on customized implementations of .NET Core. When you develop apps using these technologies, you may not be aware that you are taking advantage of .NET Core:
 
-- ASP.NET Core.  ASP.NET Core is a modular version of ASP.NET that combines ASP.NET MVC and the ASP.NET web API. It runs on both the .NET Framework and .NET Core and is designed for building high-performance cloud and microservice-style applications; it is not intended as a replacement to ASP.NET in the .NET Framework.
+- ASP.NET Core.  ASP.NET Core is a modular version of ASP.NET that combines ASP.NET MVC and the ASP.NET web API. It runs on both the .NET Framework and .NET Core and is designed for building high-performance cloud and microservice-style applications; it is not intended as a replacement to ASP.NET in the .NET Framework. For information on ASP.NET Core, see [Introduction to ASP.NET 5](http://docs.asp.net/en/latest/conceptual-overview/aspnet.html "Introduction to ASP.NET 5").
```

Comment:
see [Introduction to ASP.NET Core] 
Title has changed in dev branch and will change today about 11 AM. You can see it now at https://docs.asp.net/en/dev/conceptual-overview/aspnet.html